packages/frontend-2/plugins/060-dataPreload.ts

```diff
@@ -30,11 +54,35 @@ export default defineNuxtPlugin(async (ctx) => {
       queries: [
         { query: activeUserQuery },
         { query: mainServerInfoDataQuery },
+        { query: serverInfoBlobSizeLimitQuery },
+        { query: navigationProjectInvitesQuery },
         ...(isWorkspacesEnabled.value
           ? [
+              ...(isBillingEnabled.value
```

Comment:
w/ SSR logs in browser console its a lot easier to see what queries can be optimized by preloading